Automattic/woocommerce-payments
# Start webhook listener on server in E2E setup script

#### Changes proposed in this Pull Request

Starts up the webhook listener during `npm run test:e2e-setup` so that behavior that relies on them can work and be tested. For instance, the Transactions (and Deposits) lists will populate:

<img width="1274" src="https://user-images.githubusercontent.com/1867547/124192897-0642fb00-da94-11eb-8410-6f8ef7dc32af.png">

The new command is identical to what's found in the server's `listen-to-webhooks.sh` script, except that it runs in detached mode (with `-d` flag) so that the script execution isn't blocked.

(Should this command have its own `step` above it?)

#### Testing instructions





* Run `npm run test:e2e-setup` and `npm run test:e2e`, and verify that webhook-dependent effects are present (such as Transactions table being filled).

-------------------
